utils_for_test.go

```diff
@@ -84,3 +85,8 @@ func writeFile(t *testing.T, prefix, contents string) string {
 func getTestTracer(serviceName string) (opentracing.Tracer, io.Closer) {
 	return jaeger.NewTracer(serviceName, jaeger.NewConstSampler(true), jaeger.NewNullReporter())
 }
+
+func getTestLogger() *zap.Logger {
+	logger := zap.NewNop()
+	return logger
```

Comment:
can inline, in fact it might not be worth having this function unless we precreate one and use it everywhere.